src/Athena/Views/Home/Home.cshtml

```diff
@@ -47,14 +47,50 @@ else
 
 }
 
+@Html.Partial("_CurrentScheduleModal", false)
+@Html.Partial("_UnmetDependencyError")
+
+<div id="modal2" class="modal">
```

Comment:
Let's make this ID a little more descriptive...

Reply:
changed to remove-class-modal